### PR TITLE
added CONTRIBUTION.md

### PR DESCRIPTION
**Pull request by DCoder-1**
## Reference to the issue ✔️

The following PR is in reference to the "adding CONTRIBUTION.md file #23 " issue. 

## Proposed Changes ✔️

Added a small guide for the beginners for their first-time contribution help.

## Commands that you used to complete this issue ✔️

- git add .
- git commit -m "message"
- git push origin main

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+where `this-is-you` is your GitHub username. Here you're copying the contents of the first-contributions repository on GitHub to your computer.
+
+## Create a branch
+
+Change to the repository directory on your computer (if you are not already there):
+
+```
+cd DSA
```
Comment (sohan2410) [inline CONTRIBUTING.md:43]: can you please change from `cd DSA` to `cd GitFest`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+git push origin <add-your-branch-name>
+```
+
+replacing `<add-your-branch-name>` with the name of the branch you created earlier.
+
+## Submit your changes for review
+
+If you go to your repository on GitHub, you'll see a `Compare & pull request` button. Click on that button.
+
+<img style="float: right;" src="https://user-images.githubusercontent.com/68538660/106243186-ebafe180-61d6-11eb-8fbc-87ef0b8839b6.png" alt="create a pull request" />
```
Comment (sohan2410) [inline CONTRIBUTING.md:94]: Can you take ss of GitFest repo and attach, same for to submit the pull request

**Comment by DCoder-1** [comment]
hey @sohan2410, I have attached the screenshots for this repo and made the other stated changes as well.

**Comment by sohan2410** [comment]
LGTM, great work @DCoder-1! Please update the PR description and we are good to merge🚀

**Comment by DCoder-1** [comment]
Hey @sohan2410 please check it out.

**Pull request merged**